cql3/Cql.g

```diff
@@ -555,6 +564,7 @@ updateConditions returns [conditions_type conditions]
  * USING TIMESTAMP <long>
  * WHERE KEY = keyname
    [IF (EXISTS | name = value, ...)];
+ * [USING (TIMESTAMP <timestamp> | TIMEOUT <duration>) [AND ...]]
```

Comment:
Above we have `USING TIMESTAMP <long>`, here `TIMESTAMP <timestamp>` - is the difference intentional? If not, we should probably use the same one.

Reply:
`long` is more correct as the timestamp value is indeed a pure number and not a cql timestamp (i.e. millis since epoch), thanks.

Reply:
fixed in v4